providers/sadad/models.go

```diff
@@ -0,0 +1,118 @@
+package sadad
+
+import (
+	"github.com/GoFarsi/paygap/client"
+	"time"
+)
+
+type MultiplexingType int64
+
+const (
+	Percentage MultiplexingType = iota
+	Amount
+)
+
+type Sadad struct {
```

Comment:
Base struct better use for keep API configuration and Auth to external service, please remove **amount** and etc.

Reply:
Thank you,
Done.